blackjack.py

```diff
@@ -106,14 +107,18 @@ def get_soft_hand_value(self):
         return (21.1 if self.check_natural_win()
                 else sum(card.points for card in self.hand))
 
-    def get_hard_hand_value(self):
+    def get_hand_value(self):
```

Comment:
Now that's we've improved the implementation, let's talk about conventions.
While getters and setters are common in Java-like languages, Python doesn't really do that. So seeing a `get_*` method on a class should raise some eyebrows. Doesn't mean it's bad, just that we should look into it.

Right now, to get the hand value we do:
```
player.get_hand_value()
```
But maybe it'd be as good if not better to be able to do:
```
player.hand_value
```

This is not a required change, but it'll make you practice (and maybe learn) a common Python pattern, so I'd suggest at least trying it. I'm talking about [properties](https://www.programiz.com/python-programming/property).
In your case, you could change your code to:
```
    @property
    def hand_value(self):
        if self.check_natural_win():
            return 21.1
        else:
            hand_total = self.get_soft_hand_value()
            if hand_total <= 11 and any(card.value == 'Ace' for card in self.hand):
                hand_total += 10
            return hand_total
```

Reply:
done